app/lib/dttp/params/placement_assignment.rb

```diff
@@ -116,6 +117,18 @@ def third_subject
 
         { "dfe_ITTSubject3Id@odata.bind" => "/dfe_subjects(#{course_subject_id(trainee.course_subject_three)})" }
       end
+
+      def funding_params
+        return {} unless send_funding_to_dttp? && trainee.training_initiative != ROUTE_INITIATIVES_ENUMS[:no_initiative]
+
+        {
+          "dfe_initiative1id_value" => "/dfe_initiatives(#{training_initiative_id(trainee.training_initiative)})",
+        }
+      end
+
+      def send_funding_to_dttp?
+        FeatureService.enabled?("show_funding") && FeatureService.enabled?("send_funding_to_dttp")
```

Comment:
```suggestion
FeatureService.enabled?(:show_funding) && FeatureService.enabled?(:send_funding_to_dttp)
```

We've got a mixture of strings/symbols, I think we should stick with symbols unless the property is nested?